decidim-proposals/spec/events/decidim/proposals/rejected_proposal_event_spec.rb

```diff
@@ -42,4 +42,41 @@
       expect(subject.resource_text).to eq translated(resource.answer)
     end
   end
+
+  describe "notification digest mail" do
```

Comment:
See the comments left for `decidim-proposals/spec/events/decidim/proposals/accepted_proposal_event_spec.rb`